docs/.vuepress/config.js

```diff
@@ -1,141 +1,140 @@
 module.exports = {
-  title: "Tendermint Documentation",
-  description: "Documentation for Tendermint Core.",
-  ga: "UA-51029217-1",
-  dest: "./dist/docs",
-  base: "/docs/",
-  markdown: {
-    lineNumbers: true
-  },
+  theme: "cosmos",
+  // locales: {
+  //   "/": {
+  //     lang: "en-US"
+  //   },
+  //   "/ru/": {
+  //     lang: "ru"
```

Comment:
?

Reply:
currently there is no russian translation so commented out the options